Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -1,44 +1,44 @@
 ---
 TillRegex: !nestedRegex
-  def: (?<till>\b(to|till|til|until|thru|through)\b|{BaseDateTime.RangeConnectorSymbolRegex})
+  def: (?<till>\b(tot|totdat|gedurende|tijdens|ten tijde van)\b|{BaseDateTime.RangeConnectorSymbolRegex})
   references: [ BaseDateTime.RangeConnectorSymbolRegex ]
-RangeConnectorRegex : !nestedRegex
-  def: (?<and>\b(and|through|to)\b|{BaseDateTime.RangeConnectorSymbolRegex})
+RangeConnectorRegex: !nestedRegex
+  def: (?<and>\b(en|tot en met|t\\m|tot|tot aan)\b|{BaseDateTime.RangeConnectorSymbolRegex})
   references: [ BaseDateTime.RangeConnectorSymbolRegex ]
+ArticleRegex: !simpleRegex
+  def: \b(de|het|een)\b
 RelativeRegex: !simpleRegex
-  def: \b(?<order>dit|volgende|komende|aankomende|aanstaande|deze|huidige|vorig|verleden|voorgaande|vorige|laatste)\b
+  def: \b(?<order>dit|deze|volgend[e]?|komend[e]?|aankomend[e]?|aanstaand[e]?|huidig[e]?|vorig[e]?|verleden|voorgaand[e]?|vorig[e]?|laatste|afgelopen)\b
```

Comment:
@tellarin @STayinloves any preference between using `(e)?` vs `[e]?`? When it applies to a single character, like the example above. Afaik performance and functionality is the same.

Reply:
`(e)` catch a group while `[e]` doesn't. I think just `e?` is sufficient.